lib/stateToHTML.js

```diff
@@ -427,19 +442,19 @@ var MarkupGenerator = function () {
               if (styleSet.has(_styleName)) {
                 var _inlineStyles$_styleN = _this.inlineStyles[_styleName];
                 var _element = _inlineStyles$_styleN.element;
-                var _attributes3 = _inlineStyles$_styleN.attributes;
+                var _attributes2 = _inlineStyles$_styleN.attributes;
                 var _style = _inlineStyles$_styleN.style;
 
                 if (_element == null) {
                   _element = 'span';
                 }
                 // Normalize `className` -> `class`, etc.
-                _attributes3 = (0, _normalizeAttributes2.default)(_attributes3);
+                _attributes2 = (0, _normalizeAttributes2.default)(_attributes2);
                 if (_style != null) {
                   var styleAttr = (0, _styleToCSS2.default)(_style);
-                  _attributes3 = _attributes3 == null ? { style: styleAttr } : _extends({}, _attributes3, { style: styleAttr });
+                  _attributes2 = _attributes2 == null ? { style: styleAttr } : _extends({}, _attributes2, { style: styleAttr });
                 }
-                var attrString = stringifyAttrs(_attributes3);
+                var attrString = stringifyAttrs(_attributes2);
```

Comment:
Because I dont use npm publish, instead I refer to a github commit. So the compiled version has to be in the source. Is there a better way to do it?